orttraining/orttraining/test/training_api/common/synthetic_data_loader.h

```diff
@@ -109,7 +92,7 @@
   // did not explicitly copy the data in.
   // And also, the created OrtValue also won't clean the raw data pointer. The raw data should be removed when
   // the life time of this struct ends.
-  std::vector<std::unique_ptr<SyntheticSampleBatch>> sample_batch_collections_;
```

Comment:
>std::unique_ptr<SyntheticSampleBatch [](http://example.com/codeflow?start=14&length=36)

Can we use std::optional? See coding standards.

Reply:
I think I already removed the std::unique_ptr in the latest code. :)